<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_searchFIle</class>
 <widget class="QDialog" name="Dialog_searchFIle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>791</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>791</width>
     <height>431</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>789</width>
      <height>429</height>
     </rect>
    </property>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>791</width>
       <height>431</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_selectionImg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>371</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecionar Imagem</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>500</y>
     <width>371</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
